src/platform/meteorlake/include/platform/lib/memory.h

```diff
@@ -56,7 +56,7 @@
 /**
  * size of HPSRAM system heap
  */
-#define HEAPMEM_SIZE 0x40000
+#define HEAPMEM_SIZE 0xD0000
```

Comment:
Btw, not sure if this is really used now with Zephyr RTOS config. e.g. it should come from Zephyr, but this could still be on the TODO list.

Reply:
@lgirdwood ACE and IMX are still using HEAPMEM_SIZE from the headers. For cAVS, we are taking the values from linker scripts. All variants are in sof/zephyr/lib/alloc.c